ql/pom.xml

```diff
@@ -362,6 +370,12 @@
       <groupId>org.apache.orc</groupId>
       <artifactId>orc-tools</artifactId>
       <version>${orc.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-client-api</artifactId>
```

Comment:
you excluded in the parent pom, I don't think you need to exclude everywhere now

Reply:
orc-tools is in ql/pom.xml only. We don't have in parent pom.